tests/foundationals/latent_diffusion/test_schedulers.py

```diff
@@ -14,7 +14,13 @@ def __call__(self, x: Tensor, noise: Tensor, step: int) -> Tensor:
         def _generate_timesteps(self) -> Tensor:
             return Tensor()
 
-    scheduler = DummyScheduler(10, 20, 0.1, 0.2, "cpu")
+    scheduler = DummyScheduler(
```

Comment:
Perhaps test_scheduler_utils can be turned into parametric just to cover the uniform/quadratic/karras options?